carla-carma-integration/carla_carma_bridge/CMakeLists.txt

```diff
@@ -8,7 +8,10 @@ project(carla_carma_bridge)
 find_package(catkin REQUIRED COMPONENTS
   autoware_msgs
   rospy
+  cav_srvs
   cav_msgs
+  health_monitor
```

Comment:
Is this dependency on health_monitor necessary? I don't see it being used anywhere